usaspending_api/api_contracts/contracts/v2/references/submission_periods.md

```diff
@@ -0,0 +1,49 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Custom Account Data [/api/v2/download/accounts/submission_periods]
```

Comment:
The URL needs to match the intended URL and the title is from the original contract
```suggestion
# DABS Submission Window Dates [/api/v2/references/submission_periods/]
```